grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -661,8 +660,10 @@ public void delete() {
         clearGraph();
         txCache().closeTx(ErrorMessage.CLOSED_CLEAR.getMessage());
 
+        //TODO We should not hit the REST endpoint when deleting keyspaces through a graph
+        //TODO retrieved from and EngineGraknGraphFactory
```

Comment:
Stylistic issue:

Not 

```
//TODO line 1
//TODO line 2
```

rather

```
//TODO line 1
              line 2
```

Otherwise multiple todos are registered on the IDE when there is really only 1 todo